composer.json

```diff
@@ -12,12 +12,13 @@
         }
     ],
     "require": {
+        "ext-json": "*",
```

Comment:
I would advise to remove this extension as it is already existing in other packages as well.

Reply:
We should not rely on other packages' dependencies if we are using `json_*` functions in this package's code.

Check [this](https://dev.to/timoschinkel/good-practices-when-working-with-composer-5a1c#:~:text=To%20root%20or%20not%20to%20root) for more details

Reply:
@rennokki let me know what you think, thanks.

Reply:
I got encouraged by PhpStorm to add this package as well in https://github.com/renoki-co/php-k8s/pull/380 but didn't. I'd also recommend putting it explicitly in the requires section.